app/assets/stylesheets/earthworks.css

```diff
@@ -152,3 +152,15 @@ Uncomment lines 142-144
     display: none;
   }
 }
+
+.pagination .page-link {
```

Comment:
The PR description doesn't seem to match the changes here.  Is this the right commit?